src/scripts/main.js

```diff
@@ -2,7 +2,17 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableBody = document.querySelector('tbody');
 
-// write your code here
+people.forEach(person => {
+  const personRow = `<tr>
+  <td>${person.name}</td>
+  <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
+  <td>${person.born}</td>
+  <td>${person.died}</td>
+  <td>${person.age = person.died - person.born}</td>
+  <td>${person.century = Math.ceil(person.died / 100)}</td>
```

Comment:
Avoid modifying the 'person' object. Calculate the century directly in the template string instead of assigning it to 'person.century'.